src/pages/Setting/components/Inquiry.tsx

```diff
@@ -0,0 +1,25 @@
+import styles from './Inquiry.module.scss';
+
+export default function Inquiry(): JSX.Element {
+  return (
+    <div>
+      <div className={styles.header}>
+        <div className={styles.header__title}>문의하기</div>
+        <div className={styles['header__sub-title']}>쩝쩝박사에게 궁금한 점이 있나요?</div>
+      </div>
```

Comment:
조금은 의미론적인 태그를 사용해주면 어떨까요?
그리고 `font-weight`가 빠진 것 같아요.
```suggestion
      <header className={styles.header}>
        <h1 className={styles.header__title}>문의하기</h1>
        <h3 className={styles['header__sub-title']}>쩝쩝박사에게 궁금한 점이 있나요?</h3>
      </header>
```

Reply:
수정했습니다!
3c121e6c145f1f618f9ca9ed865a3248fb09a553